Cargo.toml

```diff
@@ -29,10 +30,10 @@ azure-devops = { project = "cobalt-org", pipeline = "liquid-rust" }
 
 [features]
 default = ["extra-filters", "serde"]
-extra-filters = []
+extra-filters = ["liquid-lib/extra-filters"]
```

Comment:
Getting rid of it entirely or just remove it from default?

Reply:
Completely.

The plan is for `liquid` to not expose any plugins.  It will only expose ways to activate the stdlib plugins.  If anyone wants to customize beyond that, they need to depend on liquid_lib (or other crates) and pull them in themselves.

Reply:
Ok, so Cobalt will add `liquid-lib` as a dependency with appropriate features